BlockSettleUILib/Trading/RFQDialog.cpp

```diff
@@ -214,7 +214,8 @@ std::shared_ptr<bs::SettlementContainer> RFQDialog::newCCcontainer()
 {
    try {
       ccSettlContainer_ = std::make_shared<ReqCCSettlementContainer>(logger_
-         , signContainer_, armory_, assetMgr_, walletsManager_, rfq_, quote_, xbtWallet_, fixedXbtInputs_, utxoReservationManager_, std::move(ccUtxoRes_));
+         , signContainer_, armory_, assetMgr_, walletsManager_, rfq_, quote_, xbtWallet_
+         , fixedXbtInputs_, utxoReservationManager_, std::move(ccUtxoRes_), armory_->topBlock());
```

Comment:
Not sure this is a good idea to pass topBlock at construction - I guess it may change during the TX creation. You have the `armory_` already passed to container - so you can safely call `topBlock()` inside.

Reply:
Sure, fixed, thanks.